doc/theory/glossary.tex

```diff
@@ -287,6 +287,22 @@
 	
 	The name is inspired from the case when the persistence diagram comes from persistent homology.
 	
+	\subsection*{Heat kernel:} \label{heat kernel}
+	
+	Given a set $\xi$, $k:\xi \times \xi \rightarrow \mathbb{R}$ is said to be a \textit{kernel} if there exists a Hilbert space $H$ with inner product $\langle \cdot, \cdot \rangle$ and a function $\phi: \xi \rightarrow H$, such that $k(x,y) = \langle \phi(x), \phi(y) \rangle$.
+	
+	In contrast to vectorisation methods, kernel methods do not give an explicit feature map. Nevertheless, such methods remain popular for tasks like classification.
+	
+	Let $D, D'$ be two \hyperref[persistence diagram]{persistence diagrams}. Then, the \textit{persistence scale-space kernel}, often called the \textit{heat kernel}, is defined by
+	\begin{equation*}
+	k_\sigma(D,\, D') = \frac{1}{8\pi\sigma}\sum_{p\in D,\, q \in D'}
+	\exp\left(\frac{-\Vert p-q\Vert^2}{8\sigma}\right) -
```

Comment:
I think we want to enclose the two \exp between parentheses since the sum applies to both.